### PR TITLE
Fix deprecated notice when calling trim

### PR DESCRIPTION
**Pull request by AbdiTolesa**
Ticket URL: https://secure.helpscout.net/conversation/2599522042/199655

### CODE REVIEW

File: classes/helpers/FrmAppHelper.php [hunk 1]
```diff
@@ -1999,9 +1999,11 @@ public static function checked( $values, $current ) {
...
 
 	public static function check_selected( $values, $current ) {
-		$values  = self::recursive_function_map( $values, 'trim' );
-		$values  = self::recursive_function_map( $values, 'htmlspecialchars_decode' );
-		$current = htmlspecialchars_decode( trim( $current ) );
+		$values = self::recursive_function_map( $values, 'trim' );
+		$values = self::recursive_function_map( $values, 'htmlspecialchars_decode' );
+		if ( $current ) {
+			$current = htmlspecialchars_decode( trim( $current ) );
+		}
```
Comment (truongwp) [inline classes/helpers/FrmAppHelper.php:2006]: `trim( null )` returns `''`. So I think we should add an `else` here and assign `$current = ''`.

Comment (AbdiTolesa) [reply classes/helpers/FrmAppHelper.php:2006]: I didn't get it, we are not calling `trim()` on `null` at all now right?

Comment (truongwp) [reply classes/helpers/FrmAppHelper.php:2006]: If it's `null`, we should set it to `''`.

**Review by truongwp — APPROVED** [review]
This looks good. Thanks @AbdiTolesa!

**Pull request merged**